tests/filesystem.go

```diff
@@ -374,12 +375,8 @@ func removeEmpty(strings []string) []string {
 	return r
 }
 
-func generateUUID(t *testing.T, ctx context.Context) (string, error) {
-	out, err := run(t, ctx, "uuidgen")
-	if err != nil {
-		return "", err
-	}
-	return strings.TrimSpace(string(out)), nil
+func generateUUID(t *testing.T) (string, error) {
```

Comment:
Can remove the `t` argument

Reply:
Actually this entire function can be dropped in favor of https://godoc.org/github.com/pborman/uuid#New

Reply:
In the future, I'd say do the replacement for all calls to this in a seperate commit, but it'd be hard to seperate now and is pretty minor, so I'd leave it for now.